pom.xml

```diff
@@ -1371,6 +1371,9 @@
             <testFailureIgnore>${maven.test.failure.ignore}</testFailureIgnore>
             <junitxml>.</junitxml>
             <argLine>${argLine} ${extraJavaTestArgs}</argLine>
+            <systemProperties>
+              <log4j.configurationFile>file:src/test/resources/log4j2.properties</log4j.configurationFile>
```

Comment:
not all modules have `src/test/xxx` structure, see module `gluten-iceberg/`.

Can we avoid duplicating this configuration file by just setting it like `${project.basedir}/xxx` in root pom?

Reply:
I haven't tested w/ a single root `log4j2.properties`, so I'm not sure whether it will work or not. But copying it module by module is also a good practice that follows the behavior of the Spark project. It allows us to customize each copy for different logging behaviors of different modules, or w/ different log4j2 dependencies.

Reply:
ok. Please note some modules' structure is like `src-iceberg/test/resources/...`.

Reply:
Maybe, do some refactor on the directory name? See previous discussion: https://github.com/apache/incubator-gluten/pull/8870#issuecomment-2696899361